test/functional/fixtures/runner/upload/upload.test.js

```diff
@@ -0,0 +1,19 @@
+'@fixture Upload';
+'@page ./index.html';
+
+
+'@test'['Upload single file test - should fail'] = {

```

Comment:
Upload a single file - should fail
--or--
Upload a non-existent file - should fail